Reaktoro/Equilibrium/SmartEquilibriumSolver.cpp

```diff
@@ -531,8 +551,65 @@ struct SmartEquilibriumSolver::Impl
         return result;
     }
 
+    auto solve(ChemicalState& state, double T, double P, VectorConstRef be, Index istep, Index icell) -> SmartEquilibriumResult
```

Comment:
We should have this `SmartEquilibriumSolver` impl independent of time step and and mesh cell location. What are `istep` and `icell` used for?

Reply:
Agree. This was added just for testing purposes